include/morphio/errorMessages.h

```diff
@@ -9,35 +9,50 @@
 #include <morphio/mut/section.h>
 
 namespace morphio {
-/**
-   Set the maximum number of warnings to be printed on screen
-**/
+/** Set the maximum number of warnings to be printed on screen **/
 void set_maximum_warnings(int n_warnings);
+/** Set whether to interpet warning as errors **/
 void set_raise_warnings(bool is_raise);
+/** Set a warning to ignore **/
 void set_ignored_warning(Warning warning, bool ignore = true);
+/** Set an array of warnings to ignore **/
 void set_ignored_warning(const std::vector<Warning>& warning, bool ignore = true);
-
+/** print a warning **/
```

Comment:
It's named `printError`, though?

Reply:
Yes but it prints warnings :D, I'm here only to document.

Reply:
```suggestion
/** Print a warning. Raises an error if  `set_raise_warnings` was set to `true`. **/
```

Reply:
Ah, thanks, that makes more sense!